### PR TITLE
PR : Feature/auth registration UI

### LINKED ISSUE #7: Prepare Registration screen UI.
Choose the registration Design and create this UI in your app :)

### PR DESCRIPTION
**Pull request by RobinKim-SWEngineer**
## What is it about?
 - Registration UI using the e-mail address

## How is it written?
 - Main widget is a Stack, which has two Containers in it. One container is for background gradient, and the second one has five input frames with a registration button. 

## How to test it?
 - Each input frame has a **different keyboard** type, with the password getting **obscured** as the user puts it in.

## Sample image?
sample_

### CODE REVIEW

File: lib/Screens/AuthRegistration/body.dart [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/cupertino.dart';
```
Comment (AndreyKalashnik) [inline lib/Screens/AuthRegistration/body.dart:1]: why didn't you exported material?

File: lib/Screens/AuthRegistration/body.dart [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  _BodyState createState() => _BodyState();
+}
+
+class _BodyState extends State<Body> {
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: [
+        Container(
+          height: double.infinity,
```
Comment (AndreyKalashnik) [inline lib/Screens/AuthRegistration/body.dart:57]: Probably, here you can use the SizedBox instead https://api.flutter.dev/flutter/widgets/SizedBox-class.html

**Review by AndreyKalashnik — CHANGES_REQUESTED** [review]
Good start!

File: lib/Screens/AuthRegistration/body.dart (hunk 1 above)
Comment (RobinKim-SWEngineer) [reply lib/Screens/AuthRegistration/body.dart:1]: Imported!

File: lib/Screens/AuthRegistration/body.dart [hunk 3]
```diff
@@ -73,25 +74,55 @@ class _BodyState extends State<Body> {
...
+                Space(20),
                 _buildInputFrame(
                   label: 'Password',
                   inputHint: 'Enter your Password',
                   shouldInputBeObscured: true,
+                  inputType: TextInputType.visiblePassword,
                   prefixIcon: Icons.lock,
-                )
+                ),
+                Space(20),
```
Comment (AndreyKalashnik) [inline lib/Screens/AuthRegistration/body.dart]: I checked your **Space** widget, and it takes one parameter and return SizedBox with the same parameter. Why don't we use just SizedBox, to be more clear ? :)

File: lib/Screens/AuthRegistration/body.dart [hunk 4]
```diff
@@ -73,25 +74,55 @@ class _BodyState extends State<Body> {
...
+                _buildInputFrame(
```
Comment (AndreyKalashnik) [inline lib/Screens/AuthRegistration/body.dart:111]: I like how you made code modular by chunks!

**Review by AndreyKalashnik — CHANGES_REQUESTED** [review]

File: lib/Screens/AuthRegistration/body.dart (hunk 3 above)
Comment (RobinKim-SWEngineer) [reply lib/Screens/AuthRegistration/body.dart]: Refactoring done! 👍

**Review by AndreyKalashnik — APPROVED** [review]
Aaand done! Merging.... =)

**Pull request merged**